### ISSUE TITLE
Feature request: Upload to my imgur account?

### ISSUE THREAD

**Issue by ptim** [FigBug/imguru#3]
Love it, thanks, works a charm - is it possible to UL directly into my account?

**Comment by KloudAlpha** [comment]
> I will probably do this, don't know when.

hey its been a decade, any update? The new imgur content policy will purge photos uploaded without an account

**Comment by FigBug** [comment]
Yeah, I probably won't get to this any time soon. I was a lot more complex than I though.
